doc/source/serve/http-guide.md

```diff
@@ -1,13 +1,36 @@
-# Calling Deployments via HTTP
+# HTTP with Serve
 
 
+
+This section should help you understand how to:
+
+- configure your HTTP server location
+- send HTTP requests to Serve deployments
+- use Ray Serve to integrate with FastAPI
+- use customized HTTP Adapters
+
+
+## Configuring HTTP Server Locations
+
+By default, Ray Serve starts a single HTTP server on the head node of the Ray cluster.
+You can configure this behavior using the `http_options={"location": ...}` flag
+in {mod}`serve.run <ray.serve.run>`:
+
+- **HeadOnly**: start one HTTP server on the head node. Serve
+  assumes the head node is the node you executed `serve.run`
+  on. This is the default.
+- **EveryNode**: start one HTTP server per node.
+- **NoServer** or `None`: disable HTTP server.
```

Comment:
sorry, I think you misunderstood my previous comment, this section should be totally dropped now because it's no longer configurable. We simply always go with the `EveryNode` behavior.

Reply:
so let's get rid of that section and just specify that behavior as a top-level point